### DISCUSSION TITLE
Turn Radius - Tracked Vehicles

### DISCUSSION THREAD

**Discussion by Scottvdken** [Courseplay/Courseplay_FS22#2736] [Ideas]
2-track tractors consistently jackknife themselves with implements that don't use a 3pt. In FS19 CP I could use the turn radius adjustment to limit the angle. Is it possible to bring this back?

**Comment by Tensuko** [comment]
Instead of bring back another thing ppl have to adjust, could you give us an example of what vehicle have the problem at wich point?
Becouse I haven't seen this issue in fs22 yet so it might be a problem of the tool not reporting its correct turn radius.

**Comment by Scottvdken** [comment]
Happens with multiple tools, but could be I suppose. I have tested with both the  Fendt 1100 and a 9RT. Jackknifes the big JD planter, a Samson lime/manure spreader, and a Demco grain auger.  Oh, jacknifed the largest base-game mulcher also. Even skipping 2 rows at a time, it still jacknifes and gets itself stuck. 

I haven't tested articulated tractors much, but the turn radius adjustment used to help with those in '19 also. 

In '19 anyways, you didn't have to adjust it unless you wanted to (same as Work Width), so it shouldn't be an inconvenience to anyone who doesn't want to mess with it.

**Comment by Tensuko** [comment]
Don't see a problem here. Setting won't be added for this.
![fsScreen_2023_08_30_18_57_13](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/c0354f28-6bc5-491c-9733-d27dab718be5)
![fsScreen_2023_08_30_18_48_22](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/00f6b217-6b75-47a1-9c44-4140ce4cdaa9)
![fsScreen_2023_08_30_18_53_27](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/b984ba11-0949-4755-b318-5c9c190a8d86)
![fsScreen_2023_08_30_18_52_21](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/852e74ac-ea7d-4130-b572-3b93b7085b29)
![fsScreen_2023_08_30_18_49_52](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/65301447-c1e0-486e-bc67-57dbf8062a5e)

**Comment by Tensuko** [comment]
If you don't like the parameter we get from the vehicle, you can always create your own personal vehicle configuration file.
Everything is explained here:
https://github.com/Courseplay/Courseplay_FS22/wiki/The-VehicleConfigurations.xml-file.#what-is-this-file-for

**Reply by S-eler** [reply]
@Tensuko  sorry for dig something from the past - but how can I find proper vehiclefilename.xml for base game stuff, to configure it?

**Reply by Tensuko** [reply]
You can use our console command for that. Sit in the vehicle and type it into the console.
![grafik](https://github.com/user-attachments/assets/93d2db8a-e0e4-4623-a57a-19eead01b59e)
